### PR TITLE
feat(input file): create input file component

### PR DESCRIPTION
**Pull request by marinasmah**
Create new component "Input File"
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/input-file/InputFile.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import reset from "@/wc_scss/reset.scss";
+import { customElement, html, LitElement, query, property, internalProperty } from "lit-element";
+import styles from "./scss/module.scss";
+import "@/components/button/Button";
+
+import { nothing } from "lit-html";
+@customElement("md-input-file")
+export class InputFile extends LitElement {
+  @property({ type: String }) getLocalization = "Get";
```
Comment (yananym) [inline web-components/src/components/input-file/InputFile.ts:17]: Remember how you had file extensions allowed explicitly when you had this in code editor? i think i would be helpful to also have a way to limit file types here also if required (through a prop)

File: web-components/src/components/input-file/InputFile.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ */
+
+import reset from "@/wc_scss/reset.scss";
+import { customElement, html, LitElement, query, property, internalProperty } from "lit-element";
+import styles from "./scss/module.scss";
+import "@/components/button/Button";
+
+import { nothing } from "lit-html";
+@customElement("md-input-file")
```
Comment (yananym) [inline web-components/src/components/input-file/InputFile.ts:15]: maybe call it more like a "file upload" ?

**Review by yananym — CHANGES_REQUESTED** [review]
just a couple of improvements suggested

File: web-components/src/components/input-file/InputFile.ts (hunk 2 above)
Comment (marinasmah) [reply web-components/src/components/input-file/InputFile.ts:15]: Not sure. HTML Element calls as "input type file". What do you think?

File: web-components/src/components/input-file/InputFile.ts (hunk 1 above)
Comment (marinasmah) [reply web-components/src/components/input-file/InputFile.ts]: Updated

**Pull request merged**